src/tests/ports/owners.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { ISubgraphComponent } from '@well-known-components/thegraph-component'
+import { FetchOptions } from '../../ports/merger/types'
+import { createOwnersComponent } from '../../ports/owner/component'
+import {
+  IOwnerDataComponent,
+  OwnerFragment,
+  OwnersFilters,
+  OwnersSortBy,
+} from '../../ports/owner/types'
+
+let collectionSubgraphMock: ISubgraphComponent
+let getOwnersQueryMock: jest.Mock
+let ownersComponentMock: IOwnerDataComponent
+let queryFilters: FetchOptions<OwnersFilters, OwnersSortBy>
+
+describe('when fetching owners', () => {
+  beforeEach(() => {
+    getOwnersQueryMock = jest.fn()
+
+    collectionSubgraphMock = {
+      query: getOwnersQueryMock,
+    }
+
+    ownersComponentMock = createOwnersComponent({
```

Comment:
Why is this called a mock if you are using the real function to create the component? Should just be `ownersComponent`